arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -50,8 +52,11 @@ def cancel_step(self, _input: predefined_schemas.cancelInput):
 
     @plugin.step_with_signals(
         id="start-pcp",
-        name="Start PCP",
-        description="Start the PCP data logging tools",
+        name="Run PCP",
+        description=(
+            "Runs the PCP data collection and then processes the results into a "
+            "machine-readable format"
+        ),
```

Comment:
(I think Dave already commented on this, but....)  The `id` seems out of sync with the `name` and `description`, now.